### DISCUSSION TITLE
Why is heads and dhead hardcoded in flash attention test ?

### DISCUSSION THREAD

**Discussion by 3outeille** [ELS-RD/kernl#320] [Q&A] [answered]
At this [line](https://github.com/ELS-RD/kernl/blob/main/test/test_attention.py#L65), `heads` and `dhead` are hardcoded. Is flash attention only working for those specific values ? If so why is that ?

**Comment by gaetansnl** [comment, marked as answer]
@3outeille hello, we use different values in other tests. It was just arbritrary values.
